content/zh/docs/reference/config/analysis/ist0119/index.md

```diff
@@ -5,11 +5,13 @@ owner: istio/wg-user-experience-maintainers
 test: no
 ---
 
-当认证策略指定使用 JWT 认证但目标 [Kubernetes 服务](https://kubernetes.io/zh-cn/docs/concepts/services-networking/service/) 配置不正确时，会出现此消息。
-正确定位到 Kubernetes 服务需要使用 http|http2|https 前缀来命名端口
-（请参见 [协议选择](/zh/docs/ops/configuration/traffic-management/protocol-selection/)），并且还需要协议使用 TCP；协议留空也可以，因为其默认值就是 TCP。
+当认证策略指定使用 JWT 认证但目标
```

Comment:
I actually haven't seen this message before, and have no idea about what this meesage is doing, I'm going to check it.

Reply:
I think the analyzer that generates the message has been removed for a long time, see https://github.com/istio/istio/pull/22706. I think we can just retrie this message and remove the docs from the istio.io. Would you like to do this?

Reply:
See https://github.com/istio/istio/pull/28830/files#diff-5ab445fd2493bb54c4667c7e74f7c0e5773ab35bf24fc5e639ab24d90a333274R216 as an example, and you can modify https://github.com/istio/istio/blob/master/pkg/config/analysis/msg/messages.yaml#L179-L192 in the same way.

Reply:
After the message is cleaned up, the file will be synced to https://github.com/istio/istio.io/blob/master/data/analysis.yaml, then you can delete the corresponding message description in istio.io

Reply:
Well received. Will update the istio repo soon before remove ist0119 here. Thanks for your guidance 👍